src/frequenz/sdk/timeseries/_quantities.py

```diff
@@ -0,0 +1,656 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""Types for holding quantities with units."""
+
+import math
+from datetime import timedelta
+from typing import Self, TypeVar, overload
+
+GenericQuantity = TypeVar(
+    "GenericQuantity", "Quantity", "Power", "Current", "Voltage", "Energy"
+)
```

Comment:
I would rename this as `QuantityT` or `TQ` or something like that, `GenericQuantity` sounds more like an alias for `Generic[Quantity]`.

```suggestion
QuantityT = TypeVar(
    "QuantityT", "Quantity", "Power", "Current", "Voltage", "Energy"
)
```

Reply:
done